books/RayTracingTheNextWeek.html

```diff
@@ -1305,10 +1346,13 @@
                             ];
                 return perlin_interp(c, u, v, w);
             }
-            static vec3 *ranvec;
-            static int *perm_x;
-            static int *perm_y;
-            static int *perm_z;
+
+        public:
+            static const int point_count = 256;
+            static vec3* ranvec;
```

Comment:
Are we not moving `perlin` over to `vector`s and `shared_ptr`s?

Reply:
No, there's no need. See comment above.